### PR TITLE
feat: 🚀 Add default color to PolygonZoneAnnotator, drawing functions

### PR DESCRIPTION
**Pull request by DivyaVijay1234**
# Description

Default Colors Added: Default colors were added to utility functions such as drawLine, drawRectangle, drawPolygon, and PolygonZoneAnnotator.
Documentation: Documentation was added to explain the changes and usage of the new default colors.

Fix: #1587

## Type of change

- New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: supervision/draw/utils.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 
 def draw_line(
-    scene: np.ndarray, start: Point, end: Point, color: Color, thickness: int = 2
+    scene: np.ndarray, start: Point, end: Point, color: Color=Color(163, 81, 251), thickness: int = 2
```
Comment (onuralpszr) [inline supervision/draw/utils.py:12]: Please use our color scheme in like "sv.Color.WHITE"  for all them. You can find them in here 

https://github.com/roboflow/supervision/blob/337b0e4a5b5df9617ecd166a008b92472526bd2f/supervision/draw/color.py#L231

**Comment by onuralpszr** [comment]
@DivyaVijay1234  please accept CLA and please check my review and after that we can move on thank you.

**Comment by LinasKo** [comment]
Hi @DivyaVijay1234 👋 

Thank you for the contribution!

> Which of those colours would you like me to use as the default colour?

The default color we prefer is `Color.ROBOFLOW`. I see you set the values for it, but that's in fact not necessary. You can simply replace `Color(r, g, b)` with `Color.ROBOFLOW`.

**Comment by DivyaVijay1234** [comment]
@onuralpszr @LinasKo 
Thank you for guidance. Please confirm if I need to make any further changes or if the issue has now been solved.

**Comment by onuralpszr** [comment]
> @onuralpszr @LinasKo Thank you for guidance. Please confirm if I need to make any further changes or if the issue has now been solved.

push code please :)

**Comment by DivyaVijay1234** [comment]
Please check if the changes I made are now visible. Thank you.

**Comment by onuralpszr** [comment]
@DivyaVijay1234  I made some small documentation additions and also add white color to other annotator you missed with documentations, thank you for your contribution I am merging in this and close the issue as complete as well. 

cc @LinasKo

**Comment by onuralpszr** [comment]
I made some fixes on PR title and comment part to be more descriptive so that everyone else can understand what is going on in here as well.

**Review by onuralpszr — APPROVED** [review]
LGTM !

**Comment by DivyaVijay1234** [comment]
@onuralpszr Thank you!

**Comment by LinasKo** [comment]
@onuralpszr , the documentation updates are helpful, but I'm confused why PolygonAnnotator got white as the default color. Does this also pick it for text boxes? Otherwise, the line should be roboflow-colored.

**Comment by onuralpszr** [comment]
> @onuralpszr , the documentation updates are helpful, but I'm confused why PolygonAnnotator got white as the default color. Does this also pick it for text boxes? Otherwise, the line should be roboflow-colored.

The reason I used white because I wanted to follow LineZoneAnnotator also using white lines so I wanted to follow that too 

https://supervision.roboflow.com/latest/detection/tools/line_zone/#supervision.detection.line_zone.LineZoneAnnotator-functions

```python
__init__(thickness=2, color=Color.WHITE, text_thickness=2, text_color=Color.BLACK, text_scale=0.5, text_offset=1.5, text_padding=10, custom_in_text=None, custom_out_text=None, display_in_count=True, display_out_count=True, display_text_box=True, text_orient_to_line=False, text_centered=True) 
```

**Comment by LinasKo** [comment]
@onuralpszr , that makes sense! I did not notice that. Thank you for the guidance and PR resolution!

**Comment by onuralpszr** [comment]
> @onuralpszr , that makes sense! I did not notice that. Thank you for the guidance and PR resolution!

Pleasures 🙏 ☺️

**Pull request merged**